### ISSUE TITLE
[Data] Crypto response attributes missing for some symbols

### ISSUE THREAD

**Issue by MorningSleeper** [iexg/IEX-API#1313]
I notice some supported crypto symbols, for example ETHUSD, are missing the following response attributes.

bidPrice
bidSize
askPrice
askSize

![image](https://user-images.githubusercontent.com/8920632/68293346-9a9c5680-0042-11ea-8c55-0b6da9ccde0b.png)

Shouldn't they appear in JSON response of all supported crypto symbols even if blank?

API URL: https://cloud.iexapis.com/v1/crypto/ETHUSD/quote?token=XXX

Time: Nov 6th 2019 3:08 PST

PS - You probably recall I reported the same [issue](https://github.com/iexg/IEX-API/issues/1281#issue-508124642) for a different crypto pair LTCBTC but the issue seemed to resolved itself before you got a chance to look into it so you closed the ticket.

**Comment by devbanana** [comment]
Yes I'd love to see this resolved as well. Currently the BTCUSD pair is next to useless besides getting price information as nothing else is showing up, either missing or null.

**Comment by MorningSleeper** [comment]
The originally reported issue still exists. The following response attributes

bidPrice
bidSize
askPrice
askSize

do not show up in JSON output by endpoint for reported URL For other crypto pairs the response attributes do show up and have a non null value. Seems the above response attributes should be output in JSON even if they have a value of null, no?

**Comment by MorningSleeper** [comment]
The originally reported issue still exists for the supported symbol ETHUSD.
